### ISSUE TITLE
Deprecated methods consistency

### ISSUE THREAD

**Issue by ykazakov** [owlcs/owlapi#993]
While porting some OWL API 4 code to OWL API 5 I noticed some (inconsistent) method deprecation, e.g., `HasRemoveAxiom#removeAxiom(OWLOntology ont, OWLAxiom axiom)` has been deprecated in favour of `OWLOntology#removeAxiom(OWLAxiom)`, but not the symmetrical method `HasAddAxiom#addAxiom(OWLOntology ont, OWLAxiom axiom)`, even though, of course, there is also `OWLOntology#addAxiom(OWLAxiom)`. I guess the latter method should be also deprecated.

**Comment by ignazio1977** [comment]
Indeed, `HasAddAxiom#addAxiom(OWLOntology ont, OWLAxiom axiom)` should be deprecated.
